src/server.js

```diff
@@ -20,12 +21,17 @@ app.use(bodyParser.json());
 // Request logging
 app.use(pino({ logger }));
 
-// Add routes to the app.
-// app.get('/', (req, res) => res.send('User Data Information API!'));
 app.use(routes());
 
-// Swagger for documenting the api, access through localhost:xxxx/api-docs.
-app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
+$RefParser.dereference(swaggerDocument, (err, schema) => {
```

Comment:
$ Should be removed to streamline naming conventions of constants. According to above change.